index.js

```diff
@@ -1,5 +1,103 @@
 // Calculator project
-const display = document.querySelector('.display');
-const one = document.querySelector('.one');
+display = document.querySelector('.display');
+let result = ''
+let completed = false
 
-const total = 0;
+let number = [];
+for (var i = 0; i <= 9; ++i) {
+    number[i] = document.querySelector(".number" + i);
+    number[i].addEventListener("click", function () { addDigit() });
+}
+
+remove = document.querySelector('.delete');
```

Comment:
leaving out variable keywords is only possible when javascript is not in `strict` mode. I'll talk about this during the session.